jxl/src/frame/quantizer.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) the JPEG XL Project Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+use crate::{
+    bit_reader::BitReader,
+    error::{Error, Result},
+    headers::encodings::{Empty, UnconditionalCoder},
+};
+
+#[derive(Debug)]
+#[allow(dead_code)]
+pub struct LfQuantFactors {
+    pub quant_factors: [f32; 3],
+    pub inv_quant_factors: [f32; 3],
+}
+
+impl LfQuantFactors {
+    pub fn new(br: &mut BitReader) -> Result<LfQuantFactors> {
+        let mut quant_factors = [0.0f32; 3];
+        if br.read(1)? == 1 {
+            quant_factors[0] = 1.0 / 4096.0;
+            quant_factors[1] = 1.0 / 512.0;
+            quant_factors[2] = 1.0 / 256.0;
```

Comment:
I suppose one could set
const DEFAULT_QUANT_FACTORS: [f32; 3] = [1.0 / 4096.0, 1.0 / 512.0, 1.0 / 256.0];
and then copy_from_slice or something, also looks good as is...

Reply:
IMO it's clear enough as it is :-)